boot/zephyr/Kconfig

```diff
@@ -364,6 +370,50 @@ config BOOT_WAIT_FOR_USB_DFU
 	  for USB DFU to be invoked. Please note DFU always updates the
 	  slot1 image.
 
+config BOOT_USB_DFU_WAIT_DELAY_MS
+	int
+	depends on BOOT_USB_DFU_WAIT
+	default 12000
+	help
+	  The duration to wait for USB DFU to be invoked.
+
+config BOOT_USB_DFU_GPIO
+	bool "Use GPIO to detect whether to trigger DFU mode"
+	select USB
+	select USB_DFU_CLASS
+	select IMG_MANAGER
+	help
+	  If y, MCUboot uses GPIO to detect whether to invoke USB DFU.
+	  Please note DFU always updates the slot1 image.
+
+endchoice
+
+config BOOT_USB_DFU_DETECT_PORT
+	string "GPIO device to trigger USB DFU mode"
+	depends on BOOT_USB_DFU_GPIO
+	default GPIO_0 if SOC_FAMILY_NRF
+	help
+	  Zephyr GPIO device which contains the pin used to trigger
```

Comment:
"contains the pin (...)" is restrictive clause, which identifies a specific GPIO, use "that" instead of "which".